openlibrary/templates/default_edit.html

```diff
@@ -170,6 +170,7 @@
     </form>
 </div>
 
+$# FIXME: Someone needs to work out what on earth all this JS is for and whether we can remove it.
```

Comment:
Based a quick bit of archaeology, I believe a possible ancestor is  https://code.google.com/archive/p/repetitionmodel/ which implemented the WHATWG Web Forms 2.0 spec as a cross-browser shim and which was replaced by https://code.google.com/archive/p/webforms2/. Today it can presumably be all ripped out and replaced with HTML5 Forms.

Reply:
@mekarpeles @jdlrobson See above concerning repetition model.